### DISCUSSION TITLE
how to collect metric when concourse run with docker image

### DISCUSSION THREAD

**Discussion by PhilippeAccorsi** [concourse/concourse#7444] [General]
Hi,

We start to use Concourse-ci and we use concouse with the concourse docker image. But I not find documentation about how connect my prometheus server to the concourse server to collect metric.

Where I can find information about that ?
Is it possible to collect metric directly on a http port or I need to develop an exporter ?

Thanks for your help.
Philippe

**Comment by taylorsilva** [comment]
Not all config options are documented. You can see all config options by running: `docker run -t concourse/concourse web --help`.
For Prometheus:
```
Metric Emitter (Prometheus):
--prometheus-bind-ip=       IP to listen on to expose Prometheus metrics. [$CONCOURSE_PROMETHEUS_BIND_IP]
--prometheus-bind-port=     Port to listen on to expose Prometheus metrics. [$CONCOURSE_PROMETHEUS_BIND_PORT]
```
Does that give you enough info?
You can also see this: https://github.com/concourse/concourse/blob/master/hack/overrides/prometheus.yml

**Comment by PhilippeAccorsi** [comment]
Thanks for your answer.

I need to check this solution. I come back here to explain if everything its ok for me.

**Comment by PhilippeAccorsi** [comment]
Thanks a lot, thats working perfect :)
I just need to find a good grafana dashboard now (or to make/improve existing one ^^ ).
